backend/test/routes/model/getModel.spec.ts

```diff
@@ -0,0 +1,21 @@
+// Some bad parts
+import '../../../src/utils/test/testUtils.js'
+
+import { describe, expect, test, vi } from 'vitest'
+
+import { getModelSchema } from '../../../src/routes/v2/model/getModel.js'
+import { createFixture, testGet } from '../../../src/utils/v2/test/routes.js'
+
+describe('routes > model > getModel', () => {
```

Comment:
Still no mocked parse function? I think we agreed to mock it for unit tests and write system integration tests that would include the parse function (zod functionality) and the middleware